From fe4373675c0f8a4946fbdd1828770495e78a0164 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Sun, 21 Jan 2018 17:22:17 +0200
Subject: [PATCH 2208/2241] mvebu: defconfig: Increase the VA address space to
 48 bit

The A8K+ address map requires support for memory addressed
up to 0x8100000000. This maxumum address is defined by
MAX_MEMBLOCK_ADDR constant that is derived from VA_BITS.
The default VA_BITS value is 39 bits, which allows addressing
memory up to 0x4000000000.
This patch moves the VA_BITS to the next supported value of 48 bits.

Change-Id: I80e7afa25c2603b29d9e0f072496ac63a3cd17da
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/49295
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index c9b5a32..6c26c3b 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -40,6 +40,7 @@ CONFIG_PCIE_IPROC=y
 CONFIG_PCIE_ARMADA_8K=y
 CONFIG_PCI_AARDVARK=y
 CONFIG_PCIE_A8K_DW_EP=y
+CONFIG_ARM64_VA_BITS_48=y
 # CONFIG_MMIO_64BIT is not set
 CONFIG_SCHED_MC=y
 CONFIG_PREEMPT=y
-- 
2.7.4

